File: Qin/Qin-Android/app/src/main/java/edu/bfa/ss/qin/Util/Student.java
package edu.bfa.ss.qin.Util;

public class Student {
    public String LCObjectID;
    public String SchoolID;
    public String Advertising;
    public String BaiduFaceID;
    public int ID;
    public String Name;

    public byte[] getStudentBeaconMinor() {
        byte[] bytes = {0x00, 0x00};
        bytes[0] = (byte)(ID >> 8 & 0xFF);
        bytes[1] = (byte)(ID & 0xFF);
        return bytes;
    }
}

File: QinDevice/QinDevice-Mac/QinDevice-Mac/Student.swift
//
//  Student.swift
//  Qin-ios
//
//  Created by 杨璨 on 2020/1/6.
//  Copyright © 2020 canyang. All rights reserved.
//

import Foundation

public class Student {
    public var LCObjectID: String = ""
    public var SchoolID: String = ""
    public var Advertising: String = ""
    public var BaiduFaceID: String = ""
    public var ID: Int = 0
    public var Name: String = ""
    
    public func getStudentBeaconMinor() -> [UInt8] {
        var bytes:[UInt8] = [0x00, 0x00]
        bytes[0] = (UInt8)(ID >> 8 & 0xFF)
        bytes[1] = (UInt8)(ID & 0xFF)
        return bytes
    }
}
